app/assets/javascripts/discourse/app/templates/modal/sidebar-section-form.hbs

```diff
@@ -22,67 +22,30 @@
         </div>
       {{/if}}
     </div>
-    {{#each this.activeLinks as |link|}}
-      <div class="row-wrapper">
-        <div class="input-group">
-          <label for="link-name">{{i18n
-              "sidebar.sections.custom.links.icon.label"
-            }}</label>
-          <IconPicker
-            @name="icon"
-            @value={{link.icon}}
-            @options={{hash maximum=1}}
-            class={{link.iconCssClass}}
-            @onlyAvailable={{true}}
-            @onChange={{action (mut link.icon)}}
-          />
-          {{#if link.invalidIconMessage}}
-            <div class="icon warning">
-              {{link.invalidIconMessage}}
-            </div>
-          {{/if}}
-        </div>
-        <div class="input-group">
-          <label for="link-name">{{i18n
-              "sidebar.sections.custom.links.name.label"
-            }}</label>
-          <Input
-            name="link-name"
-            @type="text"
-            @value={{link.name}}
-            class={{link.nameCssClass}}
-            {{on "input" (action (mut link.name) value="target.value")}}
-          />
-          {{#if link.invalidNameMessage}}
-            <div class="name warning">
-              {{link.invalidNameMessage}}
-            </div>
-          {{/if}}
-        </div>
-        <div class="input-group">
-          <label for="link-url">{{i18n
-              "sidebar.sections.custom.links.value.label"
-            }}</label>
-          <Input
-            name="link-url"
-            @type="text"
-            @value={{link.value}}
-            class={{link.valueCssClass}}
-            {{on "input" (action (mut link.value) value="target.value")}}
-          />
-          {{#if link.invalidValueMessage}}
-            <div class="value warning">
-              {{link.invalidValueMessage}}
-            </div>
-          {{/if}}
-        </div>
-        <DButton
-          @icon="trash-alt"
-          @action={{action "deleteLink" link}}
-          @class="btn-flat delete-link"
-          @title="sidebar.sections.custom.links.delete"
-        />
+    <div class="row-wrapper header">
+      <div class="input-group link-name">
+        <label for="link-name">{{i18n
```

Comment:
The value of the `for` attribute is the id of another element but I don't think we're setting `link-name` as an element's ID anywhere.

Reply:
Also this should probably be `link-icon`.